features/correction/__init__.py

```
@@ -1,5 +1,18 @@
+)
 from .schemas import CorrectionOutput, CorrectionStatus
 
-__all__ = ["CorrectionOutput", "CorrectionStatus"]
+__all__ = [
+    "CorrectionOutput",
+    "CorrectionStatus",
+    "CorrectionRepository",
+    "get_correction_repository",
+    "init_correction_repository",
+    "reset_correction_repository",
+]
```

Ruff (0.15.2)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)